src/Peachpie.Library/Hash.cs

```diff
@@ -2818,12 +2818,12 @@ public static PhpValue password_hash(string password, PasswordType algo, PhpArra
         /// <summary>
         /// Verifies that a password matches a hash.
         /// </summary>
-        public static bool password_verify(string password, string hash)
+        public static bool password_verify(Context ctx, string password, string hash)
```

Comment:
`Context ctx` is not needed.

Reply:
Resolved https://github.com/peachpiecompiler/peachpie/pull/593/commits/6f1865f58f6166fda85c7352d98cab2d049e5ddf